### PR TITLE
Attempt to fix compiler out of heap error during build

### PR DESCRIPTION
**Pull request by codemzs**

### CODE REVIEW

File: tools/ci_build/build.py [hunk 1]
```diff
@@ -1266,7 +1266,7 @@ def openvino_run_onnx_tests(build_dir, configs, onnx_test_data_dir,
             model_dir_opset8, "test_*"))
         model_dir_opset10 = os.path.join(model_dir, "opset10")
         model_dir_opset10 = glob.glob(os.path.join(
-            model_dir_opset10, "*v1*"))
+            model_dir_opset10, "tf_*"))
```
Comment (jywu-msft) [inline tools/ci_build/build.py:1269]: Do not revert this change. This is only for openvino build which is unrelated to training

Comment (codemzs) [reply tools/ci_build/build.py:1269]: I understand but this is just a quick check, lets wait for the builds to come out.

Comment (jywu-msft) [reply tools/ci_build/build.py:1269]: Please take a look at the code. If it's true that the heap error only happens in training builds there is no way this change is related. It is only called when build.py is invoked with --use_openvino

Comment (jywu-msft) [reply tools/ci_build/build.py:1269]: Besides. I just saw that the training CI fails during link. What does that have to do with tests that run post build.

Comment (codemzs) [reply tools/ci_build/build.py:1269]: I did read the code, it is just a "try" to quickly rule out stuff. Because of these errors we lost a whole day today and while bunch of us are trying to figure out, I just want to do my part even though I may be naive since I'm fairly new to the team and work on training side. Please excuse me. 

I agree with you. Thanks.

Comment (codemzs) [reply tools/ci_build/build.py:1269]: Just to be clear, I wasn't planning on checking this in after your expressed concerns. I would still need a sign-off.
